CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Unreleased
 #### Features
+-Introduced `LocationPuckOptions` that can be used to specify location puck appearance in each of the different navigation state. [#6574](https://github.com/mapbox/mapbox-navigation-android/pull/6574)
+-:warning: Introduced `ViewStyleCustomization.locationPuckOptions` in favor of `ViewStyleCustomization.locationPuck` that can be used to leverage `LocationPuckOptions` to change the location puck at runtime for each navigation state. [#6574](https://github.com/mapbox/mapbox-navigation-android/pull/6574)
```

Comment:
I think the first line can be removed, because the second one includes it.

Reply:
Fixed.